pytorch_lightning/core/lightning.py

```diff
@@ -158,19 +164,19 @@ def optimizers(
         # multiple opts
         return opts
 
-    def lr_schedulers(self) -> Optional[Union[Any, List[Any]]]:
+    def lr_schedulers(self) -> Optional[Union[_LRScheduler, List[_LRScheduler]]]:
```

Comment:
```suggestion
    def lr_schedulers(self) -> Optional[Union[LRSchedulerTypeUnion, List[LRSchedulerTypeUnion]]]:
```
 can be ReduceLROnPlateau?